src/adapter/src/catalog/migrate.rs

```diff
@@ -179,8 +189,8 @@ pub(crate) async fn migrate(
 /// statements. This preps for the eventual removal of these options from the `CREATE SOURCE`
 /// statement.
 fn ast_rewrite_create_subsource_options(
-    tx: &mut Transaction,
     stmt: &mut Statement<Raw>,
+    all_items_and_statements: &Vec<(Item, Statement<Raw>)>,
```

Comment:
I would honestly add a comment saying to remove this after we remove your migration.